### PR TITLE
add legend

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Add legend to fieldset

### How to review

See that tests pass and code makes sense 

### Who can review

!me

### CODE REVIEW

File: assets/templates/partials/filter.tmpl [hunk 1]
```diff
@@ -90,61 +100,72 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (lindenmckenzie) [inline assets/templates/partials/filter.tmpl:162]: Is this whitespace necessary?

File: assets/templates/partials/filter.tmpl [hunk 2]
```diff
@@ -209,15 +229,15 @@
...
+                <div id="collapsible-content" class="ons-collapsible__content ons-js-details-content ons-u-mb-s">
                     {{ range $index, $filter := $value }}
                     <div class="ons-checkboxes__items">
                         <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
                             <span class="ons-checkbox ons-checkbox--no-border {{ $key }}">
                                 <input type="checkbox" id="{{ $key }}-group-{{ $index }}"
                                     class="ons-checkbox__input ons-js-checkbox"
                                     name="{{ $key }}"
-                                    value="{{ print $filter.Type }}"
+                                    value="{{ print $filter.Type }}" aria-controls="{{ $key }}-group-{{ $index }}-other-wrap"
```
Comment (lindenmckenzie) [inline assets/templates/partials/filter.tmpl:240]: and here for unneeded aria-controls.

**Review by lindenmckenzie — DISMISSED** [review]

**Comment by lindenmckenzie** [comment]
https://github.com/ONSdigital/dp-frontend-search-controller/pull/233 <-- see this.

File: assets/templates/partials/filter.tmpl [hunk 3]
```diff
@@ -90,61 +100,72 @@
...
-                                                                value="{{ $id }}"
-                                                            />
-                                                            <label class="ons-checkbox__label" for="{{ $id }}"
-                                                                id="{{ $id }}-label">
-                                                                {{ localise $childFilter.LocaliseKeyName $lang 4 }} ({{ $childFilter.NumberOfResults }})
-                                                            </label>
-                                                        </span>
-                                                    </span>
-                                                </div>
+                <fieldset id="collapsible-content" class="ons-collapsible__content ons-js-details-content ons-u-mb-s">
```
[the remark this answers is not in the document]
Comment (NoelleDL) [reply assets/templates/partials/filter.tmpl:103]: Oh shoot. Will fix this!

**Review by nshumoogum — APPROVED** [review]
LGTM

**Pull request merged**